apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/DexController.java

```diff
@@ -648,9 +671,11 @@ public Response getConfig( @Context HttpServletRequest req) throws NotFoundExcep
         tradingViewConfigDTO.supports_time = false;       
         // resolutions
         tradingViewConfigDTO.supported_resolutions =  new ArrayList<>();         
-        tradingViewConfigDTO.supported_resolutions.add("D");
-        tradingViewConfigDTO.supported_resolutions.add("2D");
-        tradingViewConfigDTO.supported_resolutions.add("3D");       
+        tradingViewConfigDTO.supported_resolutions.add("15");
```

Comment:
Need to extract duplicates of supported_resolutions config to separate field/contstants

Reply:
> Need to extract duplicates of supported_resolutions config to separate field/contestants
need to be careful with that. Tried to, but on the graph, those intervals remain to be invisible. 
In the future, the config will be the part of the frontend and this request will be deprecated.
Thus, will leave it as it is so that not to break the operability of the system.